backend/handlers/handlers.go

```diff
@@ -1438,6 +1439,32 @@ func (handlers *Handlers) getExchangeMoonpayBuyInfo(r *http.Request) (interface{
 	return resp, nil
 }
 
+func (handlers *Handlers) getBtcDirectInfo(r *http.Request) (interface{}, error) {
+	code := accountsTypes.Code(mux.Vars(r)["code"])
+	acct, err := handlers.backend.GetAccountFromCode(code)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Maybe check also for `acct != nil && acct.Offline() == nil && !acct.FatalError()` like we do in other places?